cmd/controller/ctrl.go

```diff
@@ -84,6 +90,17 @@ func (c *Controller) Run(threadiness int, stopCh <-chan struct{}) error {
 		glog.Error("Unable to push disk objects to etcd : ", err)
 		return err
 	}
+	//This infinite loop will help to get event details send by udev-event module
+	//event details send using channel.
+	for true {
```

Comment:
Not sure, where the stack ptr will go, to loop or to channels.  It is better to put this infinite loop in a goroutine and block for stopch channels. Also, pls try if using select statement is feasible with loop and goroutine.

Reply:
put that code in go routine